ci-operator/config/opendatahub-io/opendatahub-operator/opendatahub-io-opendatahub-operator-main.yaml

```diff
@@ -64,24 +68,31 @@ tests:
 - as: opendatahub-operator-e2e
   skip_if_only_changed: ^\.github/|^docs/|\.[mM][dD]$|^.gitignore$|^golangci|^crd-ref-docs\.config|^OWNERS$|^PROJECT$|^LICENSE$|^OWNERS_ALIASES$|^Makefile$
   steps:
-    cluster_profile: gcp-opendatahub
+    allow_best_effort_post_steps: true
+    cluster_profile: aws-opendatahub
     dependencies:
       OO_BUNDLE: opendatahub-operator-bundle
     env:
-      COMPUTE_NODE_TYPE: n2-standard-8
+      BASE_DOMAIN: openshift-ci-aws.rhaiseng.com
```

Comment:
do we need to update for "main" if the scope in the ticket is only for "rhoai"?

Reply:
I’d like to compare the results of both branches to see whether main and rhoai produce similar completion times and consistent e2e results.